### PR TITLE
stage6: libm2k: Add udev rules to cmake

### PR DESCRIPTION
**Pull request by mirceacaprioru**
Before this patch udev rules where not installed for libm2k this was
causing m2k board to not be visible inside scopy. This removes the cmake
option to not install udev rules.

Signed-off-by: Mircea Caprioru <mircea.caprioru@analog.com>

### CODE REVIEW

**Comment by mirceacaprioru** [comment]
Waiting for a review from @AlexandraTrifan

**Review by commodo — APPROVED** [review]
mystery solved :)

**Review by AlexandraTrifan — APPROVED** [review]

**Pull request merged**
